tests/Report/ReporterFactoryTest.php

```diff
@@ -51,7 +51,7 @@ public function testRegisterBuiltInReports()
 
         $builder->registerBuiltInReporters();
         static::assertSame(
-            ['checkstyle', 'gitlab', 'json', 'junit', 'txt', 'xml'],
+            ['checkstyle', 'gitlab', 'json', 'junit', 'raw', 'txt', 'xml'],
```

Comment:
we do need a schema for it
vide
```
$ ls doc
-rw-r--r--  1 d.ruminski d.ruminski  1825 Oct 20  2018 checkstyle.xsd
-rw-r--r--  1 d.ruminski d.ruminski  6314 Oct 20  2018 junit-10.xsd
-rw-r--r--  1 d.ruminski d.ruminski  1128 Oct 20  2018 schema.json
-rw-r--r--  1 d.ruminski d.ruminski  2660 Oct 20  2018 xml.xsd
```

Reply:
anything that is documented. eg, how we behave when input is multiple files?

Reply:
done, please check `doc/raw.md`